website/templates/institution.mako

```diff
@@ -8,17 +8,21 @@
     <div class="alert alert-info"><strong>NOTICE: </strong>Forks, registrations, and uploads will be temporarily disabled while the OSF undergoes a hardware upgrade. These features will return shortly. Thank you for your patience.</div>
 % endif
     <div id="inst">
-<div class="dashboard-header">
+<div class="dashboard-header" style="background: white">
```

Comment:
Avoid inline styling. How about adding a new class to `dashboard-page.css`:

```css
.dashboard-header.dashboard-header-institution {
    background: none;
}
```

Then applying

```html
class="dashboard-header dashboard-header-institution"
```